src/circleci-base/scripts/trigger-build.sh

```diff
@@ -4,34 +4,33 @@ set -u
 repo=$1
 branch=${2:-develop}
 
-export TYPE="Trigger: $repo:$branch"
-
 dir=$(echo "$repo" | tr -dc 'a-zA-Z0-9' | head -n 1)
 
 rm -fr "$dir"
 mkdir "$dir"
 
+echo "Triggering $repo"
+
 GIT_COMMIT_DESC=$(git --git-dir=/tmp/workspace/.git log --format=%B -n 1 "$CIRCLE_SHA1")
-if [[ $GIT_COMMIT_DESC == *"[DEV]"* ]]; then
-  GIT_PREFIX="[DEV]"
-elif [[ $GIT_COMMIT_DESC == *"[AUTO-PROCEED]"* ]]; then
-  GIT_PREFIX="[AUTO-PROCEED]"
+if [[ $GIT_COMMIT_DESC == *"[HOLD]"* ]]; then
```

Comment:
We can use `GIT_COMMIT_MESSAGE` so that it's consistent with the naming in other scripts.